src/renderer/react/components/nodes/listElement/NodeListElement.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { nodeShape } from 'react/shapes/node';
+import Button from 'react/components/general/button/Button';
+import { FormattedMessage } from 'react-intl';
+import { compose, pure, withProps } from 'recompose';
+import messages from './NodeListElement.messages';
+
+const enhancer = compose(
+  pure,
+  withProps(props => ({
```

Comment:
Use `withHandlers` instead, otherwise you are creating a new method every render.

Reply:
Even if I've a `pure` on top of it ?